ui/open-api-ui/src/test/java/io/smallrye/openapi/ui/IndexCreatorTest.java

```diff
@@ -89,15 +93,15 @@ void testCreateWithMultipleUrls() throws IOException {
 
         String s = new String(indexHtml);
 
-        assertTrue(s.contains("<title>SmallRye OpenAPI UI</title>"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-newspaper.css\" >"));
-        assertTrue(s.contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >"));
+        assertThat(s).contains("<title>SmallRye OpenAPI UI</title>");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"theme-newspaper.css\" >");
+        assertThat(s).contains("<link rel=\"stylesheet\" type=\"text/css\" href=\"style.css\" >");
         assertFalse(s.contains("url: '/openapi',"));
-        assertTrue(s.contains("<img src='logo.png' alt='SmallRye OpenAPI UI'"));
-        assertTrue(s.contains("dom_id: '#swagger-ui',"));
-        assertTrue(s.contains("deepLinking: true,"));
-        assertTrue(s.contains("urls: [{url: \"/api\", name: \"Production\"},{url: \"/swagger\", name: \"Default\"}],"));
-        assertTrue(s.contains("\"urls.primaryName\": 'Production',"));
+        assertThat(s).contains("<img src='logo.png' alt='SmallRye OpenAPI UI'");
+        assertThat(s).contains("dom_id: '#swagger-ui',");
+        assertThat(s).contains("deepLinking: true,");
+        assertThat(s).contains("urls: [{url: \"/api\", name: \"Production\"},{url: \"/swagger\", name: \"Default\"}],");
+        assertThat(s).contains("\"urls.primaryName\": 'Production',");
```

Comment:
It's difficult to review when nothing really changed. If you feel strongly that assertj-core should be used, maybe introduce that as a separate PR ?